receiver/mysqlreceiver/metadata.yaml

```diff
@@ -175,6 +188,42 @@ metrics:
       monotonic: true
       aggregation: cumulative
     attributes: [page_operations]
+  mysql.table.io_waits:
+    enabled: true
+    description: The total count of I/O waits events for a table.
+    unit: 1
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+    attributes: [io_waits_operations, io_waits_name, schema]
+  mysql.table.io_waits.time:
+    enabled: true
+    description: The total time of I/O waits events for a table.
+    unit: ms
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+    attributes: [io_waits_operations, io_waits_name, schema]
+  mysql.index.io_waits:
+    enabled: true
+    description: The total count of I/O waits events for an index.
+    unit: 1
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+    attributes: [io_waits_operations, io_waits_name, schema, io_waits_index]
+  mysql.index.io_waits.time:
```

Comment:
I think we should change these metric names a bit, based on a few things:

`io` is commonly used namespace, so let's use `foo.io.bar` instead of `foo.io_bar`.

`.count` is commonly appended in order to avoid collisions between namespaces and metrics.

Based on these, I think we should change these to:
- `mysql.table.io.wait.count`
- `mysql.table.io.wait.time`
- `mysql.index.io.wait.count`
- `mysql.index.io.wait.time`